Resources/doc/index.rst

```diff
@@ -436,7 +436,23 @@ This ignores the tables, and any named objects such as sequences, on the DBAL le
 Note that if you have multiple connections configured then the ``schema_filter`` configuration
 will need to be placed per-connection.
 
+Migrations Metadata Storage
```

Comment:
This is more a troubleshooting than a chapter for the metadata storage. Maybe you can rewrite it to make it sound more like a troubleshooting.

Reply:
```suggestion
Troubleshooting out of sync metadata storage issue
```